tox.ini

```diff
@@ -21,7 +29,7 @@ commands = pre-commit run --all-files --show-diff-on-failure
 deps =
     mypy==1.5.1
     pytest==7.4.2
-commands = mypy --show-error-codes {posargs:scrapy_spider_metadata tests}
+commands = mypy --check-untyped-defs --show-error-codes {posargs:scrapy_spider_metadata tests}
```

Comment:
if I'm not mistaken, --show-error-codes is a default in recent mypy

Reply:
would it make sense to move --check-untyped-defs to pyproject.toml, so that all mypy config is in a single place?